src/components/Search.jsx

```diff
@@ -1,5 +1,29 @@
-const Search = () => {
-  return <div>Search</div>;
+"use client";
+import { BiSearch } from "react-icons/bi";
+
+const Search = ({ value, onChange, placeholder, classes }) => {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const searchText = e.target[0].search.value;
```

Comment:
this can just be the value prop

Reply:
I've addressed the other problems on this issue. If I change it to a value prop it would track every key press from the user but why would we need that level of control? The search function from the hackathon site works similarly to what is being used here, capturing the text after 'enter' is pressed.